lib/htmx/rails/version.rb

```diff
@@ -2,7 +2,7 @@
 
 module Htmx
   module Rails
-    VERSION = '0.1.0'
-    HTMX_VERSION = '1.0.2'
+    VERSION = '0.2.0'
```

Comment:
Is it ok to change minor version here?

Reply:
Yeah, I think it's ok. We should merge your CI changes first, right now it is failing